README.md

```diff
@@ -78,6 +78,12 @@ archive.directory('subdir/', false);
 // append files from a glob pattern
 archive.glob('file*.txt', {cwd:__dirname});
 
+// append a file with a callback for when the queue has been processed.
+// usefull for implementing backpressure prevention mechanisms
+archive.file("file1.txt", { name: "file4.txt" }, () => {
```

Comment:
[✓] creating this documentation is an excellent touch
[   ] it should be moved from the `archive.file` example to the `archive.append` example(s)